pkgs/applications/graphics/megapixels/default.nix

```diff
@@ -26,20 +27,23 @@ let
 in
 stdenv.mkDerivation rec {
   pname = "megapixels";
-  version = "0.16.0";
+  version = "1.0.1";
 
   src = fetchFromSourcehut {
     owner = "~martijnbraam";
     repo = "megapixels";
     rev = version;
-    sha256 = "0z7sx76x18yqf7carq6mg9lib0zbz0yrd1dsg9qd6hbf5niqis37";
+    sha256 = "0k9a5dpr5z0g7ngbhk4j22sbs1ffxiwg8wmbzgggdc9xvwmkgppr";
   };
 
   nativeBuildInputs = [ meson ninja pkg-config wrapGAppsHook ];
 
-  buildInputs = [ gtk3 gnome3.adwaita-icon-theme zbar ]
-  ++ optional tiffSupport libraw
-  ++ optional jpgSupport graphicsmagick;
+  buildInputs = [
+    epoxy
+    gnome3.adwaita-icon-theme
```

Comment:
Is that still correct with the bump to GTK4? I'm not that knowledgable about the GTK & GNOME world rn.

I only have a non-NixOS PinePhone OS, and adding that icon theme helped with making some icons show up properly. I don't know if fixing it like that was necessary / desirable when using a mobile NixOS installation.

Reply:
Actually, packages aren't supposed to bundle icon themes: https://nixos.org/manual/nixpkgs/unstable/#ssec-gnome-packaging

Reply:
Oh, I didn't know that. Could you remove the icon theme as well then?

I'll build & test it on my PinePhone when I'm back from work today.